soh/soh/Enhancements/randomizer/Plandomizer.cpp

```diff
@@ -0,0 +1,1095 @@
+#include "Plandomizer.h"
+#include "soh/UIWidgets.hpp"
+#include "soh/util.h"
+#include <vector>
+#include "include/z64item.h"
+#include "objects/gameplay_keep/gameplay_keep.h"
```

Comment:
Try moving these to the C extern and see if it builds, I'm trying to start doing that for C headers